run_terraform.sh

```diff
@@ -1,11 +1,27 @@
-cd terraform
+#!/bin/sh -ex
+
+if [ -z "${BRANCH_NAME}" ]; then
+  BRANCH_NAME=$(git branch --show-current)
+fi
+echo "Using '${BRANCH_NAME}' environment"
```

Comment:
Cloud Build provides BRANCH_NAME as a [default substitution variable](https://cloud.google.com/build/docs/configuring-builds/substitute-variable-values#using_default_substitutions). Are you proposing this change for a use-case outside of Cloud Build?

Reply:
Oh, given the shell script I assumed this could be run interactively.

Reply:
In a real world scenario, you wouldn't want humans running it interactively. The [cloudbuild.yaml](https://github.com/danieldeleo/bigquerycicd/blob/prod/cloudbuild.yaml) file is what invokes the run_terraform.sh script